ext/TrixiMakieExt.jl

```diff
@@ -262,7 +262,7 @@ function iplot(pd::PlotData2DTriangulated;
 
     # On OSX, shift-command-4 for screenshots triggers a constant "up-zoom".
     # To avoid this, we remap up-zoom to the right shift button instead.
-    Makie.cameracontrols(ax.scene).attributes[:up_key][] = Makie.Keyboard.right_shift
+    Makie.cameracontrols(ax.scene).controls[:up_key][] = Makie.Keyboard.right_shift
```

Comment:
When was this API introduced - (how) do we need to update the compat bounds of Makie.jl/CairoMakie.jl etc.?

Reply:
I believe this was introduced in v0.21, so restricting Makie to v0.21 and CairoMakie to v0.12 is probably the way forward here.

Reply:
My bad - the change was in v0.20 of Makie.  That corresponds to CairoMakie v0.11 I believe.

Reply:
Could you please push the required changes to `Project.toml`, `docs/Project.toml`, and `test/Project.toml`?